common/remote/sockfile.cpp

```diff
@@ -3160,6 +3338,15 @@ class CClientStatsTable : public OwningStringSuperHashTableOf<CClientStats>
     }
 };
 
+struct OpenFileInfo
```

Comment:
Yes, room for improvement here. I thought should be hash table, but didn't want to change at this juncture.